lib/util.js

```diff
@@ -222,3 +222,22 @@ exports.parseFormats = function(info, debug) {
   formats.sort(exports.sortFormats);
   return formats;
 };
+
+
+
+/**
+ * @param {String} videoDescriptionHtml
+ * @return {String}
+ */
+exports.cleanVideoDescription = function(videoDescriptionHtml) {
+	if (!videoDescriptionHtml || videoDescriptionHtml == null) 
+		return null;
+
+	return entities.decode(videoDescriptionHtml
+		.replace('\n', ' ', 'gi')
+		.replace(/\s*\<\s*br\s*\/?\s*>\s*/gi, '\n')
+		.replace(/<\s*\/\s*p\s*>\s*<\s*p[^>]*>/gi, '\n')
+		.replace(/<.*?>/gi, '')
```

Comment:
I can't fully understand these regexes, I've just blindly ported the idea from the "clean_html" in https://github.com/rg3/youtube-dl/blob/master/youtube_dl/utils.py - I'll need some help with this one.